src/Pmi/Service/NotifyBiobankOrderFinalizeService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pmi\Service;
+
+use Pmi\Mail\Message;
+use Pmi\Audit\Log;
+
+class NotifyBiobankOrderFinalizeService
+{
+    protected $app;
+    protected $db;
+
+    public function __construct($app, $info)
+    {
+        $this->app = $app;
+        $this->info = $info;
+    }
+
+    public function sendEmails()
```

Comment:
Since the $this->info is specifically used by this method, would it make more sense to pass it in here instead of through the constructor? There's only one method right now, so it doesn't make a big difference, but I think it makes more sense for the constructor to only include the container.